rs/bitcoin/ckbtc/minter/tests/tests.rs

```diff
@@ -1393,6 +1393,28 @@ fn test_min_retrieval_amount_testnet() {
     ckbtc.refresh_fee_percentiles();
     let retrieve_btc_min_amount = ckbtc.get_minter_info().retrieve_btc_min_amount;
     assert_eq!(retrieve_btc_min_amount, min_amount);
+
+    // Test changing min_retrieve_fee when upgrade
+    let min_amount = 123_456;
+    let upgrade_args = UpgradeArgs {
+        retrieve_btc_min_amount: Some(min_amount),
+        min_confirmations: None,
+        max_time_in_queue_nanos: None,
+        mode: None,
+        kyt_principal: None,
+        kyt_fee: None,
```

Comment:
nit: since `UpgradeArgs` implements `Default`, we could leave all those fields set to `None`  as `..Default::default()`. This would help IMO readability since it would be immediately clear what is being changed.

Reply:
Good idea! Thanks for the suggestion!